source/extensions/filters/network/http_connection_manager/config.cc

```diff
@@ -522,6 +526,8 @@ HttpConnectionManagerConfig::HttpConnectionManagerConfig(
     early_header_mutation_extensions_.push_back(std::move(extension));
   }
 
+  auto srds_factory =
+      Envoy::Config::Utility::getFactoryByName<Router::SrdsFactory>(srds_factory_name);
```

Comment:
nit: move to the kScopedRoutes switch case.

Reply:
mind if I leave here? it's a pretty cheap function for the non-srds case and if you declare variables in a switch  you need braces and it screws up coverage.